src/inter_dc_communication_sender.erl

```diff
@@ -132,7 +125,7 @@ stop(timeout, State=#state{socket=Socket}) ->
 
 stop_error(timeout, State=#state{socket=Socket}) ->
     _ = gen_tcp:close(Socket),
-    {stop, error, State}.
+    {stop, normal, State}.
```

Comment:
I would probably leave a comment so that's clear to the reader.  I feel that you could go either way -- we do a lot of restarting due to connection timeouts via the supervisor by returning non-normal exits, but you don't have to do it that way.  As long as it's documented and clear.